### ISSUE TITLE
About hyper-paramter tuning

### ISSUE THREAD

**Issue by gihanpanapitiya** [syr-cn/SimSGT#5]
Did you do hyper-parameter tuning for each of the 10 random seeds?
Or did you hyper-parameter tune using a particular seed and train the model using another 10 different random seeds to get the test set predictions?

**Comment by syr-cn** [comment]
Thanks for your interest.

For each particular hyper-parameter setting, we ran the experiments on the 10 random seeds and used the average performance on the 10 seeds to select the best hyper-parameter setting. We did not do hyper-parameter tuning for each seed separately.
